packages/util-crypto/src/secp256k1/tweakAdd.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017-2021 @polkadot/util-crypto authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import BN from 'bn.js';
+import { ec as EC } from 'elliptic';
+
+import { assert, isU8a } from '@polkadot/util';
+
+const ec = new EC('secp256k1');
+const ecparams = ec.curve as {n:BN};
+
+const errors = {
+  PUBKEY_PARSE: 'Public Key could not be parsed',
+  TWEAK_ADD: 'The tweak was out of range or the resulted private key is invalid'
+};
+
+function isUint8Array (name: string, value: Uint8Array, length: number|number[]) {
```

Comment:
We can just use `isU8a` from `@polkadot/util` instead of this function. Then the use becomes `assert(isU8a(...), message)`

Reply:
yes but isU8a doesn't check bytelength, does it?

Reply:
No. But it is still a simple `assert(isU8a(x) && x.length === y, message)`

Reply:
ok sounds good, adding it